### PR TITLE
Add key documentation to YAML conf output

### PR DESCRIPTION
**Pull request by douglas-raillard-arm**
TODO: cli tool/online doc generating default conf for all SimpleMultiSrcConf subclasses in one file.

### CODE REVIEW

File: lisa/conf.py [hunk 1]
```diff
@@ -1235,6 +1243,62 @@ def from_map(cls, *args, **kwargs):
...
+            ``add_placeholder`` is used.
+        :type placeholder: object
+        """
+
+        def format_comment(key_desc):
+            comment = key_desc.help
+            if not comment:
+                return comment
+            else:
+                return comment[0].upper() + comment[1:]
```
Comment (valschneider) [inline lisa/conf.py]: Is that just a capitalization of the first letter? If so, wouldn't `return comment.title()` suffice?

Comment (douglas-raillard-arm) [reply lisa/conf.py]: title() makes all first letters uppercase, not just for the first word:
```
>>> 'foo bar'.title()
'Foo Bar'
```

Comment (valschneider) [reply lisa/conf.py]: Aaah. What about `.capitalize()`? :D

Comment (douglas-raillard-arm) [reply lisa/conf.py]: Good point, the docstring of `capitalize()` is a kind of tautology ...

**Pull request merged**